third_party/ruby/ruby-2.4.BUILD

```diff
@@ -1014,6 +1079,130 @@ cp $(location ext/digest/sha2/lib/sha2.rb) $(location lib/digest/sha2.rb)
+cp  $(location ext/psych/lib/psych/versions.rb)                 $(location lib/psych/versions.rb)
+cp  $(location ext/psych/lib/psych/omap.rb)                     $(location lib/psych/omap.rb)
+cp  $(location ext/psych/lib/psych/set.rb)                      $(location lib/psych/set.rb)
+cp  $(location ext/psych/lib/psych/nodes.rb)                    $(location lib/psych/nodes.rb)
+cp  $(location ext/psych/lib/psych/streaming.rb)                $(location lib/psych/streaming.rb)
+cp  $(location ext/psych/lib/psych/nodes/node.rb)               $(location lib/psych/nodes/node.rb)
+cp  $(location ext/psych/lib/psych/nodes/mapping.rb)            $(location lib/psych/nodes/mapping.rb)
+cp  $(location ext/psych/lib/psych/nodes/document.rb)           $(location lib/psych/nodes/document.rb)
+cp  $(location ext/psych/lib/psych/nodes/stream.rb)             $(location lib/psych/nodes/stream.rb)
+cp  $(location ext/psych/lib/psych/nodes/sequence.rb)           $(location lib/psych/nodes/sequence.rb)
+cp  $(location ext/psych/lib/psych/nodes/scalar.rb)             $(location lib/psych/nodes/scalar.rb)
+cp  $(location ext/psych/lib/psych/nodes/alias.rb)              $(location lib/psych/nodes/alias.rb)
+cp  $(location ext/psych/lib/psych/parser.rb)                   $(location lib/psych/parser.rb)
+cp  $(location ext/psych/lib/psych/class_loader.rb)             $(location lib/psych/class_loader.rb)
+cp  $(location ext/psych/lib/psych/tree_builder.rb)             $(location lib/psych/tree_builder.rb)
+cp  $(location ext/psych/lib/psych/json/ruby_events.rb)         $(location lib/psych/json/ruby_events.rb)
+cp  $(location ext/psych/lib/psych/json/tree_builder.rb)        $(location lib/psych/json/tree_builder.rb)
+cp  $(location ext/psych/lib/psych/json/stream.rb)              $(location lib/psych/json/stream.rb)
+cp  $(location ext/psych/lib/psych/json/yaml_events.rb)         $(location lib/psych/json/yaml_events.rb)
+cp  $(location ext/psych/lib/psych/coder.rb)                    $(location lib/psych/coder.rb)
+cp  $(location ext/psych/lib/psych/stream.rb)                   $(location lib/psych/stream.rb)
+cp  $(location ext/psych/lib/psych/syntax_error.rb)             $(location lib/psych/syntax_error.rb)
+cp  $(location ext/psych/lib/psych/y.rb)                        $(location lib/psych/y.rb)
+cp  $(location ext/psych/lib/psych/visitors.rb)                 $(location lib/psych/visitors.rb)
+cp  $(location ext/psych/lib/psych/deprecated.rb)               $(location lib/psych/deprecated.rb)
+cp  $(location ext/psych/lib/psych/handler.rb)                  $(location lib/psych/handler.rb)
+cp  $(location ext/psych/lib/psych/handlers/recorder.rb)        $(location lib/psych/handlers/recorder.rb)
+cp  $(location ext/psych/lib/psych/handlers/document_stream.rb) $(location lib/psych/handlers/document_stream.rb)
+cp  $(location ext/psych/lib/psych/exception.rb)                $(location lib/psych/exception.rb)
```

Comment:
I'm not very happy with listing out each copy like this, is there a better way to do this in bazel?